### ISSUE TITLE
How can i get iedname by client?

### ISSUE THREAD

**Issue by w7lau** [mz-automation/libiec61850#271]
I want to get iedname from others server by my client, how can i do that?

**Comment by Tachiguishi** [comment]
See example/iec61850_client_example2

**Comment by mzillgith** [comment]
As far as I know it is not always possible. You can only get the LD names. These may include the IED name when not using functional naming.

**Comment by w7lau** [comment]
> As far as I know it is not always possible. You can only get the LD names. These may include the IED name when not using functional naming.

Yes, we use LD names to find iedname, but we want to get iedname  from icd file by ied connection, so i try to ask this question.
